### PR TITLE
Add generation 9 English names.

### LINKED ISSUE #70: Gen 9 Pokémon are missing
Hi, is it planned to update this repo with Gen 9? Thanks in advance

### PR DESCRIPTION
**Pull request by crownedfoxes**
Fixes #70
Added all the English names for new Pokemon introduced in Scarlet/Violet.

### CODE REVIEW

**Comment by sindresorhus** [comment]
Would be great if you could add the localised versions too: https://github.com/sindresorhus/pokemon/issues/70#issuecomment-1326143032

File: data/en.json [hunk 1]
```diff
@@ -903,5 +903,108 @@
 	"Basculegion",
 	"Sneasler",
 	"Overqwil",
-	"Enamorus"
+	"Enamorus",
+  "Sprigatito",
```
Comment (sindresorhus) [inline data/en.json:907]: Incorrect indentation

**Comment by crownedfoxes** [comment]
> Would be great if you could add the localised versions too: [#70 (comment)](https://github.com/sindresorhus/pokemon/issues/70#issuecomment-1326143032)

I'll see what I can do.

**Comment by sindresorhus** [comment]
Closing for lack of activity.

**Pull request closed without merging**